.changeset/proud-ants-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@commercetools-frontend/application-shell": patch
+---
+
+PCM-2794: fix quick access to product list
```

Comment:
Bear in mind this description will go to our public changeset so we should not reference internal issues.

Perhaps we can update it to be something like this:
```
Fix QuickAccess component to make it use the right products list URL.
```

Reply:
Hi @CarlosCortizasCT What is the next step for this change? Will it be deployed with the next deploy train? Just trying to be sure

Reply:
Hi @obulaworld 

There is no automatic release train for `app-kit`, we trigger a new release whenever we have a bunch of changes that worth it.

Currently the only interesting thing is this fix. If this is something important, we can publish a new version today.

Reply:
Yeah it is important for the quick access links to work. Can we publish today?